internal/server/configuration.go

```diff
@@ -0,0 +1,14 @@
+package server
+
+import (
+	"github.com/gin-gonic/gin"
+
+	"github.com/infrahq/infra/api"
+	"github.com/infrahq/infra/internal/server/email"
+)
+
+func (a *API) GetServerConfiguration(c *gin.Context, _ *api.EmptyRequest) (*api.ServerConfiguration, error) {
```

Comment:
I feel like this should be part of the `/api/settings` endpoint, its a bit weird to have both `/api/settings` and `api/server-config`. They sound the same to me.

Reply:
one is server-specific configuration, one is org-specific settings.

Reply:
`/settings` is only for password requirements at the moment, which will be org specific also

Reply:
that's the org-specific one